instrumentation/runtime-telemetry/runtime-telemetry-java8/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java8/Classes.java

```diff
@@ -14,7 +14,13 @@
 import java.util.List;
 
 /**
- * Registers measurements that generate metrics about JVM classes. The metrics generated by this
+ * In case you enable the preview of stable database semantic conventions (e.g. by setting the
+ * {@code otel.semconv-stability.opt-in} system property to {@code database}), the metrics being
+ * exported will follow <a
+ * href="https://github.com/open-telemetry/semantic-conventions/tree/main/docs/database">the most
+ * recent database semantic conventions</a>.
```

Comment:
this change does not make sense to me, this class doesn't have anything to do with the database semconv

Reply:
i can revert it. where do we need to mention `otel.semconv-stability.opt-in`?